core/templates/dev/head/components/exploration_embed_button_modal.html

```diff
@@ -9,16 +9,10 @@
     </p>
 
     <accordion close-others="true">
-      <accordion-group heading="If your website allows JavaScript" is-open="false">
-        <div class="oppia-embed-modal-code" ng-click="selectText($event)">
-&lt;oppia oppia-id="<[explorationId]>" src="<[serverName]>"&gt;&lt;/oppia&gt;
-&lt;script src="https://cdn.jsdelivr.net/oppia/0.0.1/oppia-player.min.js"&gt;&lt;/script&gt;
-        </div>
-      </accordion-group>
-      <accordion-group heading="If your website does not allow JavaScript" is-open="false" style="margin-top: 3px;">
-        <div class="oppia-embed-modal-code" ng-click="selectText($event)">&lt;iframe src="<[serverName]>/explore/<[explorationId]>?iframed=true" width="700" height="1000"&gt;</div>
+      <accordion-group heading="If your website allows Iframe" is-open="false" style="margin-top: 3px;">
+        <div class="oppia-embed-modal-code" ng-click="selectText($event)">&lt;iframe src="<[serverName]>/embed/<[explorationId]>" width="700" height="1000"&gt;</div>
         <p>
-          <em>Note: explorations embedded using iframes will not have automatic scrolling and expanding on your webpage.</em>
+          <em>Note: You can change widht and height of the iframe to the size you want.</em>
```

Comment:
It's OK to drop this line. While we should try to accommodate multiple sizes, I think for now we should optimize to a fixed size and use that in the URL.

Having said that, can we leave all changes in this file to a future PR? Let's iterate on the /embed experience and only then, when we're fully happy with it, do a separate PR to change all the instructions accordingly.